WordPress/src/main/java/org/wordpress/android/ui/mlp/ModalLayoutPickerFragment.kt

```diff
@@ -121,6 +135,15 @@ class ModalLayoutPickerFragment : BottomSheetDialogFragment() {
                     }
                 }
         )
+
+        viewModel.selectedLayoutSlug.observe(this,
```

Comment:
NitPick: Ideally the VM would tell the view layer what to do -> so instead of passing a selection and deciding what to do with it here. We could pass a buttonsUiState with (blankPageVisible = false,... ). Wdyt?

Reply:
That is a great improvement suggestion. Fixed with dee9adae5844b2988b7ce9ee0fdfccd509138d01